test/fixtures/votes.yml

```
@@ -0,0 +1,9 @@
+vote1:
+  user_id: batman.id
+  work_id: parable.id
```

Instead of using ids, you should just pass the user and works directly. Fixtures don't like ids.

```suggestion
vote1:
  user: batman
  work: parable
```